packages/iobio-react-components/statistics-NA12878.autsome.bam-2024-09-16.json

```diff
@@ -0,0 +1,24 @@
+{
+	"statistics": {
+		"mapped_reads": 98670,
+		"mapped_reads_percentage": 0.9976,
+		"forward_strands": 49593,
+		"forward_strands_percentage": 0.5014,
+		"proper_pairs": 97490,
+		"proper_pairs_percentage": 0.9857,
+		"singletons": 235,
+		"singletons_percentage": 0.002376,
+		"both_mates_mapped": 98435,
+		"both_mates_mapped_percentage": 0.9952,
+		"duplicates": 387,
+		"duplicates_percentage": 0.003913,
+		"failed_qc": 0,
+		"first_mates": 49438,
+		"last_read_position": 52662640,
+		"paired_end_reads": 98905,
+		"reverse_strands": 49312,
+		"second_mates": 49467,
+		"total_reads": 98905,
+		"mean_read_coverage": 52
+	}
+}
```

Comment:
Demo file included for review. I can add `statistics-*.json` type files to gitignore if this looks good

Reply:
Added to gitignore

Reply:
unclear, do you want this merged or ignored?

Reply:
Sorry for confusion. This is not intended to be committed, but I did want to see if there were any opinions or changes requested on the output.
@justincorrigible any thoughts?

Reply:
LGTM! I'd have preferred camelCase keys, as per JS convention, but I'd imagine these are just reflecting iobio internals. correct?

Reply:
Yes these are straight from the iobio response

Reply:
Demo file removed